tests/integration/cli/tests/snapshot.rs

```diff
@@ -1268,3 +1268,12 @@ fn test_snapshot_quickjs() {
         .run_wasm(include_bytes!("./wasm/qjs.wasm"));
     assert_json_snapshot!(snapshot);
 }
+
+#[cfg_attr(any(target_env = "musl", target_os = "windows"), ignore)]
```

Comment:
Why are we ignoring these?

Reply:
No idea, I just noticed they were on every one of these tests, so I assumed there was a reason. Isn't there?

Reply:
I just removed the filters, let's see if it'll work.

Reply:
I assume this is why the filters are created: https://github.com/wasmerio/wasmer/actions/runs/5760653049/job/15617657063#step:26:208

Reply:
Ah yeah.... seems like we messed up the path detection here. 

Let's create a followup issue to fix this, but out of scope for this PR.

Reply:
Done.